ui/app/models/secret-engine.js

```diff
@@ -187,6 +187,10 @@ export default class SecretEngineModel extends Model {
     if (type === 'kv' && parseInt(this.version, 10) === 2) {
       fields.push('casRequired', 'deleteVersionAfter', 'maxVersions');
     }
+    // WIF secret engines
```

Comment:
GCP and Azure are next after AWS, so when you see the reference to WIF secret engines, these are the others I'm referring to.